lib/src/collect_lcov.dart

```diff
@@ -104,26 +105,23 @@ class LcovCollector {
   /// Generates and returns a coverage json file
   Future<CoverageResult<List<File>>> _getCoverageJson(
       String testFile, Directory coverageDir) async {
+    int observatoryPort = 8181; // FIXME: Should read out port instead?
     var args = [
-      "--coverage_dir=${coverageDir.path}",
-      "--package-root=${packageRoot}",
+      "--packages=${packagesPath}",
+      "--pause-isolates-on-exit",
+      "--enable-vm-service=${observatoryPort}",
       testFile
     ];
-    var result = await processSystem.runProcess(Platform.executable, args);
-    if (result.exitCode < 0) {
```

Comment:
I'm not sure if this block should be removed or not.  It seems duplicated by the CoverageResult handling below.